library/ix-dev/community/mineos/questions.yaml

```diff
@@ -0,0 +1,231 @@
+groups:
+  - name: MineOS Configuration
+    description: Configure MineOS
+  - name: User and Group Configuration
+    description: Configure User and Group for MineOS
+  - name: Network Configuration
+    description: Configure Network for MineOS
+  - name: Storage Configuration
+    description: Configure Storage for MineOS
+  - name: Resources Configuration
+    description: Configure Resources for MineOS
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  - variable: mineosConfig
+    label: ""
+    group: MineOS Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: username
+          label: Username
+          description: The username for MineOS WebUI.
+          schema:
+            type: string
+            empty: false
+            required: true
+            valid_chars: ^[a-zA-Z0-9]+$
```

Comment:
I don't think it is required but is it better to have this in single quotes to make sure nothing special happens when python converts this yaml to a dict?